platform/mbed_lib.json

```diff
@@ -145,7 +145,7 @@
         },
         "minimal-printf-enable-floating-point": {
             "help": "Enable floating point printing when using mprintf profile",
-            "value": false
```

Comment:
The issue is that a simple application like blinky with a Hello World printf will have floats in the image.
https://github.com/ARMmbed/mbed-os/blob/master/platform/source/minimal-printf/mbed_printf_implementation.c/#L599

Reply:
It would have anyway as printf will bring some float support in any case. For the minimal blinky example we can flip all the switches off.

Reply:
I think that floats should be disabled by default so we don't introduce floats in the image for all the applications that don't use them. It is not user friendly to have to explicitly disable floats in minimal-printf when you are not using floats to start with, compared with enabling floats printing if the application uses them.

As an example, using blinky baremetal with explicit call to printf on NUCLEO_L073RZ, there is an addition of 6400 bytes of Flash if floats are enabled in minimal-printf.

Reply:
Was there agreement on this - to have floats disabled by default ?

Reply:
Great !